config/dev/nfs_patch.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
```

Comment:
Is it necessary to add this file?

Reply:
Not necessary, this is a dev, I use it for my own development